python/lbann/contrib/args.py

```diff
@@ -235,11 +241,15 @@ def get_profile_args(args: argparse.Namespace) -> list[str]:
 
     """
     try:
+        caliper = args.caliper
         caliper_config = args.caliper_config
     except AttributeError:
         raise ValueError('passed arguments have not been processed by '
                          '`add_profiling_arguments`')
 
-    if caliper_config:
-        return ['--caliper_config', f'"{caliper_config}"']
+    if lbann.has_feature("CALIPER"):
```

Comment:
Can we print a warning if the user requests Caliper options but LBANN doesn't support them (to avoid silently dropping them)?